controllers/cluster_controller_test.go

```diff
@@ -19,11 +19,7 @@ package controllers
 import (
 	"testing"
 
-	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"sigs.k8s.io/cluster-api/util/conditions"
-
-	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
should we instead keep the corev1 alias which is more meaningful and to avoid possible clashes?

Reply:
Agreed, I'll use `corev1` instead.